opm/output/eclipse/RestartIO.cpp

```diff
@@ -249,8 +255,9 @@ RestartValue load( const std::string& filename,
 
         if (ecl_file_view_has_kw( file_view , key.c_str())) {
             const ecl_kw_type * ecl_kw = ecl_file_view_iget_named_kw( file_view , key.c_str() , 0 );
-            std::vector<double> data = double_vector( ecl_kw );
-            rst_value.extra[ key ] = std::move(data);
+            const double * data_ptr = ecl_kw_get_double_ptr( ecl_kw );
```

Comment:
Yes it will; but the extra keywords are written in `double`, and now with optional loading of restart  keywords there should be no need to mix the two classes of keywords.